dotnet/src/Connectors/Connectors.AzureOpenAI/Core/ClientCore.File.cs

```diff
@@ -96,28 +146,48 @@ internal async Task<IEnumerable<OpenAIFileReference>> GetFilesAsync(Cancellation
         return response.Value.ToArray();
     }
 
-    private static OpenAIFileReference ConvertToFileReference(OpenAIFileInfo fileInfo)
+    private static AzureOpenAIFileReference ConvertToFileReference(OpenAIFileInfo fileInfo)
```

Comment:
Wondering if doing this is a good idea? E.g. if I want to write code that works seamlessly between OpenAI and Azure OpenAI will this change cause problems?

Reply:
Once we rethink on the common package, this eventually will become one type.

Reply:
The new enums and the conversion have been removed for now. As agreed offline, it won't be possible to use AzureOpenAIFileService and OpenAIFileService interchangeably until a common abstraction (interface/base abstract class) and common model types shared by both services are in place.